libc/newhdrgen/tests/test_integration.py

```diff
@@ -0,0 +1,53 @@
+import subprocess
+import unittest
+from pathlib import Path
+
+
+class TestHeaderGenIntegration(unittest.TestCase):
+    def setUp(self):
+        self.output_dir = Path("tests/output")
```

Comment:
the output dir needs to be possible to set from an environment variable or command line argument if we want to use this with bazel.